Pod/Cache/GenericCoreDataStack.swift

```diff
@@ -211,6 +239,18 @@ public class GenericCoreDataStack<CoordinatorType: NSPersistentStoreCoordinator,
         }
     }
     
+    private func deleteIfExists(fileURL url: NSURL) {
```

Comment:
deleteIfExists says delete it if it exists, and if it does not, do nothing, not even log an error.   The only file that is truly required to be present is the main file.  This is because we've already found the file and found that it's incompatible.

I believe I would make deleteIfExists check for the file first, then try the delete.  If successful, return, if the delete then throws an error for some reason, I would let it propagate to the caller.  I wouldn't log anything if the file does not exist.